src/viewer.js

```diff
@@ -174,6 +173,19 @@ async function generateReport(bundleStats, opts) {
   });
 }
 
+async function generateJSONReport(bundleStats, opts) {
+  const {reportFilename, bundleDir = null, logger = new Logger(), excludeAssets = null} = opts || {};
+
+  const chartData = getChartData({logger, excludeAssets}, bundleStats, bundleDir);
+
+  if (!chartData) return;
+
+  mkdir.sync(path.dirname(reportFilename));
+  fs.writeFileSync(reportFilename, JSON.stringify(chartData));
+
+  logger.info(`${bold('Webpack Bundle Analyzer')} saved report to ${bold(reportFilename)}`);
```

Comment:
What do you think about showing that we saved a JSON report here?

```suggestion
  logger.info(`${bold('Webpack Bundle Analyzer')} saved JSON report to ${bold(reportFilename)}`);
```

I don't know if it's necessary but might be a nice additional touch ☺️